R/generics.R

```diff
@@ -498,7 +499,8 @@ setGeneric("table_inset<-", function(obj, value) standardGeneric("table_inset<-"
 
 #' @rdname table_inset
 #' @export
-setMethod("table_inset<-", "MatrixPrintForm",
+setMethod(
+  "table_inset<-", "MatrixPrintForm",
```

Comment:
@cicdguy in paticular, this change that the linter is trying to force is actively bad and I do not want it.